canonical-websites/www.ubuntu.com
# Merge beta

## Done

- Branch from master
- Merge beta-1.7.0 into branch
- Resolve conflicts

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- All code has been QA'd so it's mostly a case of making sure nothing looks broken
